diff --git a/library/Solarium/Core/Client/Adapter/Curl.php b/library/Solarium/Core/Client/Adapter/Curl.php
index 86a0d52..468af69 100644
--- a/library/Solarium/Core/Client/Adapter/Curl.php
+++ b/library/Solarium/Core/Client/Adapter/Curl.php
@@ -114,8 +114,26 @@ class Curl extends Configurable implements AdapterInterface
         $options = $this->createOptions($request, $endpoint);
 
         $handler = curl_init();
+
+// Pantheon Change
+// This change is not necessary. @todo, find out why these params break ping
+// or remove them earlier in the stack.
+if (strpos ($uri, 'ping' )) {
+$uri = substr($uri, 0, strpos($uri, "?"));
+}
+// End Pantheon Change
+
         curl_setopt($handler, CURLOPT_URL, $uri);
         curl_setopt($handler, CURLOPT_RETURNTRANSFER, true);
+
+// Pantheon Change
+curl_setopt($handler, CURLOPT_SSL_VERIFYPEER, false);
+$client_cert = '../certs/binding.pem';
+curl_setopt($handler, CURLOPT_SSLCERT, $client_cert);;
+curl_setopt($handler, CURLOPT_PORT, 449);
+// End Pantheon Change
+
+
         if (!ini_get('open_basedir')) {
             curl_setopt($handler, CURLOPT_FOLLOWLOCATION, true);
         }
