diff --git a/src/Solr/SolrHelper.php b/src/Solr/SolrHelper.php
index 9a46804..e46300b 100644
--- a/src/Solr/SolrHelper.php
+++ b/src/Solr/SolrHelper.php
@@ -441,7 +441,7 @@ class SolrHelper {
    *   otherwise FALSE.
    */
   public function pingServer() {
-    return $this->doPing(['handler' => 'admin/info/system'], 'server');
+    return $this->doPing(['handler' => 'admin/system'], 'server'); // @todo remove this change https://www.drupal.org/node/2761121
   }
 
   /**
